tsconfig.json

```diff
@@ -0,0 +1,8 @@
+{
+  "extends": "@goodeggs/tsconfig/base",
+  "compilerOptions": {
```

Comment:
Oh boy, this is missing `strict: true`, which is off by default and really important. Can you give that a try? We can pair if necessary.

Reply:
Never mind, it's enabled by default in our base config 😅: https://github.com/goodeggs/goodeggs-tsconfig/blob/1bf86835337c66591b2e5dcab10c7a635c84e819/base.json#L13.